web/html/src/branding/css/susemanager/components/buttons.less

```diff
@@ -35,6 +35,9 @@ button.is-plain,
   }
 }
 
+a[role="button"] {
```

Comment:
We probably want the underline to be present or missing regardless of whether the element is a `button` tag or `a` tag, no? In other words, I would expect 

```tsx
<a href="..." role="button" className="btn btn-default">...</a>
```

and

```tsx
<button className="btn btn-default">...</button>
```

to look and behave the same, right now I'm guessing they don't?

Reply:
No, they don’t behave if I use a tag with role=‘button’, it shows an underline on the hover state. I am not sure if I answered your question correctly.

Reply:
Yeah that was my comment exactly, we want those two to look the same, no?

Reply:
Yes